framework/test/bundles/libU/TestBundleU.cpp

```diff
@@ -0,0 +1,67 @@
+/*=============================================================================
+
+  Library: CppMicroServices
+
+  Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+  file at the top-level directory of this distribution and at
+  https://github.com/CppMicroServices/CppMicroServices/COPYRIGHT .
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=============================================================================*/
+
+#include "TestBundleUService.h"
+
+#include "cppmicroservices/BundleActivator.h"
+#include "cppmicroservices/BundleContext.h"
+
+#include <iostream>
+
+namespace cppmicroservices {
+
+struct TestBundleU : public TestBundleUService
+{
+
+  TestBundleU() {}
+  virtual ~TestBundleU() {}
+
+};
+
+class TestBundleUActivator : public BundleActivator
```

Comment:
The test bundle does not seem to need services, so those files and the related code can be removed. It even wouldn't need an activator, but you need some code to generate a bundle library...